modules/gathering-data-specific-features.adoc

```diff
@@ -58,6 +58,13 @@ ifndef::openshift-rosa,openshift-dedicated[]
 |`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel8:v<installed-version-NHC>`
 |Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO) Operator.
 
+Use this image if your NHC Operator version is less than 0.9.0. For more information, see the 'Gathering data' section for the specific Operator in the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift[Workload Availability for Red Hat OpenShift] documentation.
+
+|`registry.redhat.io/workload-availability/node-healthcheck-must-gather-rhel9:v<installed-version-NHC>`
+|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO) Operator.
```

Comment:
Does it make sense to have `Operator` and then `Operator`? Here and on line 59
```suggestion
|Data collection for the Red{nbsp}Hat Workload Availability Operators, including the Self Node Remediation (SNR) Operator, the Fence Agents Remediation (FAR) Operator, the Machine Deletion Remediation (MDR) Operator, the Node Health Check Operator (NHC) Operator, and the Node Maintenance Operator (NMO).
```

Reply:
Thanks @razo7 for your feedback. I have applied your suggestion. I have also removed the first instance of Operator in NHC.